renv.lock

```diff
@@ -86,6 +93,20 @@
       "Repository": "CRAN",
       "Hash": "976cf154dfb043c012d87cddd8bca363"
     },
+    "bslib": {
```

Comment:
not sure why these deps are being added to the `renv.lock` file.

i intentionally added `assertthat` to `Suggests` so that `secret` can be checked (R CMD check message) since `assertthat` was the only `secret` import that wasn't already in the lock file. `assertthat` only requires `tools`